main.go

```diff
@@ -20,8 +19,13 @@ import (
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-// annotationGKEPreemptibleKillerDeleteAfter is the key of the annotation to use to store the time to kill
-const annotationGKEPreemptibleKillerDeleteAfter string = "estafette.io/gke-preemptible-killer-delete-after-n-minutes"
+// annotationGKEPreemptibleKillerExpiryDatetime is the key of the annotation to use to store the expiry datetime
+const annotationGKEPreemptibleKillerExpiryDatetime string = "estafette.io/gke-preemptible-killer-expiry-datetime"
```

Comment:
Rename to `estafette.io/gke-preemptible-killer-state` to be in line with *estafette-letsencrypt-certificate* and *estafette-cloudflare-dns*